package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@ledgerhq/react-native-ledger-core",
   "description": "Ledger Core Library bindings for React Native",
-  "version": "4.18.6",
+  "version": "4.18.7",
```

Comment:
next time, please don't update the lib version. it's not how we must do it because we need to do from master and with the npm tooling (that will also tag the commit)

Reply:
ok, thanks. I see